### PR TITLE
committed base page template changes

### PR DESCRIPTION
**Pull request by DeepaliAgrwl**
Created new base page template and incorporated all the previous review comments. Please have a look on the changes and let me know your comments.

### CODE REVIEW

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/structure/footer/.content.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:primaryType="cq:Component"
+    jcr:title="Footer Container"
+    sling:resourceSuperType="wcm/foundation/components/responsivegrid"
+    componentGroup=".hidden"/>
```
Comment (rishabh-behl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/structure/footer/.content.xml:6]: add cq:HTML tag node too 
image

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/structure/header/.content.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:primaryType="cq:Component"
+    jcr:title="Header Container"
+    sling:resourceSuperType="wcm/foundation/components/responsivegrid"
+    componentGroup=".hidden"/>
```
Comment (rishabh-behl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/structure/header/.content.xml:6]: @DeepaliAgrwl : Add cq:htmlTag node too and test 
image

**Review by rishabh-behl — COMMENTED** [review]
@DeepaliAgrwl : pl address comments

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/structure/footer/.content.xml (hunk 1 above)
Comment (DeepaliAgrwl) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/structure/footer/.content.xml:6]: added

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/structure/header/.content.xml (hunk 2 above)
Comment (DeepaliAgrwl) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/structure/header/.content.xml:6]: added and tested. It is visible as the wrapper element in soruce code.

**Comment by DeepaliAgrwl** [comment]
The previous comments were:
1. remove comments from customfooterlibs.html file-done

2. remove comments from customheaderlibs.html file -done

3. In ui.content/src/main/content/jcr_root/content/experience-fragments/hclecomm/us/en/site/header/master/.content.xml:    why is this file part of checkin ? :

earlier I used the experience fragment for header and footer thus thies file was changed. But now the changes has been reverted back.

4. All other changes are related to template-type which has been created new.

**Review by rishabh-behl — APPROVED** [review]

**Comment by rishabh-behl** [comment]
ok

**Pull request merged**